contrib/README.md

```diff
@@ -0,0 +1,368 @@
+# Event Gateway on Kubernetes 
+
+This chart deploys the Event Gateway with etcd onto a Kubernetes cluster. Please note, the default instructions expect
+an existing kubernetes cluster that supports ingress, such as GKE. If your environment doesn't have ingress support
```

Comment:
Are there any instructions that users can use to test if their cluster supports ingress?

Also, here we say ingress, but in the top level Readme, we say load balancer which should be fixed.

Reply:
FIxed the top-level README but I'm not quite sure if there are examples to check for ingress enablement. From a `minikube` perspective we need to active enable the addons (e.g. `minikube addons enable ingress`), but from from hosted version of k8s I'm not sure.

I can dig into this and amend as necessary.